### PR TITLE
Chores/plot distance visualization

### PR DESCRIPTION
**Pull request by gcattan**
Review some of the examples.

- Adopt the convention "noplot" for example which are not ran on CI.
- Plot distance visualization on the doc (as we now have pyRiemann 0.6)
- Fix labels for the toys dataset comparison

### CODE REVIEW

File: pyriemann_qiskit/alias.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from pyriemann.estimation import ERPCovariances
+from pyriemann.classification import MDM
+from sklearn.pipeline import make_pipeline
+
+ERPCov_MDM = make_pipeline(ERPCovariances(estimator="lwf"), MDM())
```
Comment (qbarthelemy) [inline pyriemann_qiskit/alias.py:5]: IMO, you should not create a new module only for a two-step pipeline.

You should follow the example of `SSVEP` folder in pyriemann examples, having a subfolder `helpers`,
to put all functions that can be re-use by other examples.

**Review by qbarthelemy — CHANGES_REQUESTED** [review]

File: pyriemann_qiskit/alias.py (hunk 1 above)
Comment (gcattan) [reply pyriemann_qiskit/alias.py:5]: Ok, it makes sense. But if these pipelines are used by different folders than `MI`, I am not sure how to resolve the file dependency then. It works for pyRiemann because there is no relative import. How do we proceed here?

Comment (qbarthelemy) [reply pyriemann_qiskit/alias.py:5]: Since the pipeline `ERPCov_MDM` is written in one line, you can copy it in each example outside of `ERP` folder.

**Review by qbarthelemy — APPROVED** [review]

**Comment by gcattan** [comment]
Thx @qbarthelemy !
